packages/nextjs/src/config/types.ts

```diff
@@ -59,6 +59,15 @@ export type UserSentryOptions = {
 
   // Automatically instrument Next.js data fetching methods and Next.js API routes
   autoInstrumentServerFunctions?: boolean;
+
+  // Used to exclude certain serverside API routes or pages from being instrumented with Sentry. This option takes an
+  // array of strings or regular expressions - strings will exactly match a route. Matches are made against routes in
+  // the follwoing form:
+  // - "pages/home/index"
+  // - "pages/about"
+  // - "pages/posts/[postId]"
+  // - "pages/posts/[postId]/comments"
```

Comment:
l: what's the purpose of this list of routes?

Reply:
Just to give future readers an idea of what the option matches against.

Reply:
L: Technically, those are examples of the form, the form itself being `pages/<path/to/page/module>`. (I don't think they're a bad idea to have, though.)